src/IrcFeatures/DccChat.cs

```diff
@@ -130,8 +130,8 @@ internal override void InitWork(Object stateInfo)
 
             DccChatStartEvent(new DccEventArgs(this));
 
-            _sr = new StreamReader(Connection.GetStream(), Irc.Encoding);
-            _sw = new StreamWriter(Connection.GetStream(), Irc.Encoding);
+            _sr = new StreamReader(Connection.GetStream(), System.Text.Encoding.Default);
+            _sw = new StreamWriter(Connection.GetStream(), System.Text.Encoding.Default);
```

Comment:
Sure, I'm not disagreeing with that, my confusion is how the code worked in the first place since Encoding didn't exist on IrcFeatures and the IrcConnection wasn't passed to DccChat.

Shall I just modify the constructor of DccChat (or whatever) to allow it to use the current connection's text encoding?